app/services/affiliate-slots-units.json

```diff
@@ -28,6 +28,10 @@
     "heading": "Moana, a heroine kids and adults alike will admire and cheer for",
     "subheading": "Watch Now On Disney+",
     "link": "https://disneyplus.bn5x.net/DrrRa",
+    "links": {
+      "page": "https://disneyplus.bn5x.net/qq5qq",
```

Comment:
I think "article" might be a better name, "page" could be any page, even the search page.

Reply:
agreed. f267051